bundles/org.openhab.core.io.rest/src/main/java/org/openhab/core/io/rest/RESTConstants.java

```diff
@@ -38,4 +40,12 @@ public class RESTConstants {
      * Version 6: extended chart period parameter format (#3863)
      */
     public static final String API_VERSION = "6";
+
+    public static CacheControl cacheControl() {
+        CacheControl cc = new CacheControl();
+        cc.setNoCache(true);
+        cc.setMustRevalidate(true);
+        cc.setPrivate(true);
+        return cc;
+    }
```

Comment:
I think haveing a static final `CACHE_CONTROL` constant would be better than creating a new object every time. I also feel that the danger these setting might be modified is very low.